code/modules/clothing/glasses/glasses.dm

```diff
@@ -395,6 +395,15 @@
 	tint = 3				//to make them blind
 	prescription_upgradable = 0
 
+/obj/item/clothing/glasses/sunglasses/blindfold/fake
+    name = "tattered blindfold"
+    desc = "A see-through blindfold perfect for cheating at games like pin the stunbaton on the clown."
+    icon_state = "blindfold"
+    item_state = "blindfold"
+    prescription_upgradable = FALSE
+    flash_protect = 0
```

Comment:
Don't need these three here anymore, since they're all defined on the parent `/obj/item/clothing/glasses/sunglasses/blindfold` object. Since this is a subtype of that now it'll inherit them.
```suggestion
    desc = "A see-through blindfold perfect for cheating at games like pin the stunbaton on the clown."
    flash_protect = 0
```